src/modules/layout/layout-action-buttons.component.ts

```diff
@@ -0,0 +1,52 @@
+import { StacheNavLink } from '../nav/nav-link';
+import { Component, Input } from '@angular/core';
+import { Router } from '@angular/router';
+import { StacheLayout } from './layout';
+import { InputConverter } from '../shared';
+
+@Component({
+  selector: 'stache-layout-action-buttons',
+  templateUrl: './layout-action-buttons.component.html',
+  styleUrls: ['./layout-action-buttons.component.scss']
+})
+export class StacheLayoutActionButtonsComponent implements StacheLayout {
```

Comment:
Try this:
```
export class StacheLayoutActionButtonsComponent extends StacheNavComponent implements StacheLayout {
```
Remove the `constructor`, the `Router` import, and the `navigate` method.